### PR TITLE
Makes Disablers Bulky

### PR DESCRIPTION
**Pull request by DarnTheMarn**
## About The Pull Request

Makes disablers bulky. Made mostly due to [this PR](https://github.com/BeeStation/BeeStation-Hornet/pull/11858) that reduces disablers shot count

## Why It's Good For The Game

To put it as simple as can be, disabler does two incompatible things at once. It acts as a primary weapon for security, by being a 4 shot stam crit with a massive energy pool, while also being a sidearm that can be used in tandem with other weapons like shotguns by being stored in the backpack. In isolation either of these are acceptable, but when combined they are problematic. 

Think about it objectively. If today someone were to try to add a weapon to security that they can carry at any alert level, keep concealed in a bag, that only requires 4 shots to stamcrit, that has enough energy to stamcrit 6 people, they would be laughed out of the room.

You might ask why the size and not the ammo count or the damage? The answer is precedent. The damage and ammo count are not insane compared to other weapons, it's roughly equivalent to existing e-guns. The size is not. The only comparable weapons are the X-01 and the service pistols, both of which are head of staff exclusive items.

The problem with disablers being large items is that it encourages security to keep two primary weapons on them. When you offer this opportunity, players jump at it for obvious reasons. Look no further than the tonfa. It was proposed as an **alternative** to the baton, but instead of being an alternative it is used in _addition_ to the baton. You can keep a tonfa and a baton in your sec belt, so why not do it? It's not the players fault for taking the obviously superior path. Just the same, armory weapons are not an alternative to disablers they are an addition to them. You can freely load yourself up with all the lethal weaponry you want knowing that since your disabler fits in your bag, you're prepared for any situation that you have to take someone in non-lethally. God forbid someone realizes you can dual wield disablers.

Isn't being able to deal with threats lethally and non-lethally a good thing? Yes! That's why energy guns and non-lethal ballistics ammo exist! Use those!
 
This is getting rather long so I'll cut it short here, I'm willing to throw verbal fisticuffs with anyone about this. If people really don't like disablers being bulky I can go with idea #2 where their damage and shot count is reduced, but size is preserved, leaning into the sidearm rather than primary weapon classification.

TLDR: Disablers being large instead of bulky is outside the norm and encourages unsportsmanlike conduct
## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Screenshot 2024-12-29 090918
Screenshot 2024-12-29 090913

</details>

## Changelog
:cl:
balance: Disablers are now bulky and no longer fit in backpacks.
/:cl:

### CODE REVIEW

**Comment by Therealdoooc213** [comment]
Nobody falling for the darn bait pr? fucked up...

**Comment by DarnTheMarn** [comment]
>EDIT: I'd like to note this is literally a bait PR, I asked darn to make a PR to make drama and this is the result

It is a legit PR though, disablers are the exception and I don't see why they should be. Sawn off shotguns too but that's basically ban bait

**Comment by ClownMoff** [comment]
🔑

**Comment by Therealdoooc213** [comment]
> It is a legit PR though, disablers are the exception and I don't see why they should be. Sawn off shotguns too but that's basically ban bait

It was a originally bait... you cant lie. I asked you to make a pr to make drama and this popped out

**Comment by Fronsis** [comment]
Bait pr 🔑

**Comment by DarnTheMarn** [comment]
The fact checkers are in, STATEMENT IS FALSE!
Screenshot 2024-12-29 112517
There's more stuff I said along those lines this is just the first I could find on discord. Curious how no one is arguing the disabler isn't super potent for a small weapon though

**Comment by Therealdoooc213** [comment]
Joever

**Comment by Hardly3D** [comment]
If the disabler is to be made bulky, it should probably also reflect that visually. As it is right now, it's a bit strange that an energy sidearm is as heavy as an energy gun.

**Comment by DarnTheMarn** [comment]
I'd be willing to resprite it to reflect the size, but I won't until a maint has feedback on the change first.

**Comment by Fronsis** [comment]
The fact that this pr was made as a joke-bait, which is a recurring topic for you, makes me have no faith in your intentions of balance and it comes very malicious and feels like discussing it it's a waste of time, i'll give some small words.

Unecessary change for the time being while we're still adjusting and processing the most recent changes added(shield belts, new limb disabling values, dismembering, slowdowns, etcetera), if no one is super happy but no one super upset then that's a cue that the iconic sidearm is in a middle ground (and that was before the aforementioned new changes were in)and has yet to see full use against these recent changes, lowering the value of ammo NOW amidst all these recent changes will impact negatively in my opinion, otherwise someone with a shield belt and a sharp weapon will insta win a random officer that doesn't carry a lethal weapon, that's what i picture.

**Comment by PerishedFraud** [comment]
If you asked me "what's the issue with disablers" I would've ignored everything in this PR and zeroed in on the fact that you can dual wield them to achieve power that surpasses batons in melee (while still being ranged).

**Comment by DarnTheMarn** [comment]
> If you asked me "what's the issue with disablers" I would've ignored everything in this PR and zeroed in on the fact that you can dual wield them to achieve power that surpasses batons in melee (while still being ranged).

I wonder if the fact that disablers are large items, that fit in your bag letting you have two guns and a full sec belt, in addition to having a faster equip time and lower slowdown and less dual wield spread due to the fact they are large instead of bulky has something to do with it.

**Comment by DarnTheMarn** [comment]
Not trying to be an asshole and I'm being serious about that, but you said some very clearly stupid and wrong things here.

>if no one is super happy but no one super upset then that's a cue that the iconic sidearm is in a middle ground 

In what world is the disabler in the middle ground. It's the best non lethal ranged weapon there is. It's smaller, meaning less slowdown and quicker equips than any other sec weapon while taking 4 shots to crit, 2 shots to disable a l limb and having a massive ammo pool in addition. Like I said before, I'm not strictly opposed to security having a sidearm, but a sidearm should have a corresponding power level to reflect that. A disabler doesn't, it's better than an e gun on non lethal.

>otherwise someone with a shield belt and a sharp weapon will insta win a random officer that doesn't carry a lethal weapon, that's what i picture.

Stamina damage does jack shit to deplete shields, so what good will carrying a disabler do for you. Shield belts were mostly made in response to disablers being as good as they are.

>The fact that this pr was made as a joke-bait, which is a recurring topic for you, makes me have no faith in your intentions of balance

I didn't make this as a joke or bait PR, Dooc asked me something along the lines of where's the controversial PR, and then I made this which I had thought about making for a while as you could see by my talks with Varo and others in the discord when he made his disabler PR.

**Comment by Tsar-Salat** [comment]
So the dilemma here is, is a disabler a sidearm or a primary weapon. I've skimmed what was said a little, but haven't read all of the comments made thus far yet.

I do agree with the issue that this PR is bringing up, that the disabler is sorta in a limbo between having kinda the power of a 'primary' weapon without the full weight that primaries have. I think thats a valid thing to bring up, I saw this argument in the disabler nerf ideasguy thread on discord, so I don't think this pr is out of the blue or anything.

I do think the direction however should be going the other way of this pr. I think the majority of players would agree that the disabler is in the spirit of a sidearm, even if it doesn't *exactly* have the attributes of it.

----

I am however somewhat confused why the weight was chosen? In the poll you made, it didn't seem like there was particular support for making them bulkier.

I think this probably needs more discussion, and better canvassing to see what people think.

**Comment by DarnTheMarn** [comment]
>I do think the direction however should be going the other way of this pr. I think the majority of players would agree that the disabler is in the spirit of a sidearm, even if it doesn't exactly have the attributes of it.

I am willing to push it in that direction as I've said, I chose the weight in spite of the poll since most people said it should be kept the same, which is ridiculous. I figured lowering peoples armament on green by making disablers a true sidearm would piss them off more than just making it bigger. 



<details>
<summary>
certified-it-never-happens-but-if-it-does-its-a-good-thing moment
</summary>

c

s

m

l

</details>

**Comment by Fronsis** [comment]
I mean... yeah, what you showed in those screenshots it's the usual equipment.. you run out of lethals/main gun? you pull out your trusty **non-lethal** sidearm to try to disengage/repel/arrest them without needing to risk going melee and that feels fine and makes sense, if the issue is people getting horizontal'd too quickly then you can slightly reduce the ammo as it was proposed by Varo while leaving the disabler on it's current wight, i believe a disabler should be capable to fend off at least two people and not making it be like old PTSD that run out of ammo in 2 seconds.

**Comment by DarnTheMarn** [comment]
>you run out of lethals/main gun? 

If you can't take someone out or hold out until you can resupply with the amount of gear in the pictures, you deserve to die.

>sidearm

This is the problem. There's only two ways the disabler is a sidearm as is, the sprite and the size. If it's going to stay a sidearm it's going to be brought to service pistol level.

**Comment by DarnTheMarn** [comment]
For you people who think nerfing the disabler to be a sidearm is the better choice, [here you go](https://github.com/BeeStation/BeeStation-Hornet/pull/12117) now everyone is happy.

**Comment by PowerfulBacon** [comment]
Been thinking about this and I think it should be done alongside a leg holster accessory that gives you a single extra slot for holding pistols next to the suit slot

**Comment by DarnTheMarn** [comment]
I dont know what you mean by that, that would just make it only affect the people who steal a disabler

**Comment by PowerfulBacon** [comment]
I don't think this PR is necessary or good for the game.

**Comment by DarnTheMarn** [comment]
As long as the LP-12 pistol gets merged there's no real need for discussion, that addresses the problematic parts. The dual wield removal also helps.
